### PR TITLE
Lint PHP files with "rector/rector"

### PR DESCRIPTION
**Pull request by phansys**

### CODE REVIEW

File: src/Tree/Strategy/ORM/Nested.php [hunk 1]
```diff
@@ -608,7 +609,7 @@ public function shiftRL(EntityManagerInterface $em, $class, $first, $delta, $roo
                     continue;
                 }
 
-                $nodeMeta = $em->getClassMetadata(get_class($node));
+                $nodeMeta = $em->getClassMetadata($node !== null ? get_class($node) : self::class);
```
Comment (phansys) [inline src/Tree/Strategy/ORM/Nested.php:612]: Fix these cases. `$node` must not be `null`.

File: tests/Gedmo/Timestampable/Fixture/ArticleCarbon.php [hunk 2]
```diff
@@ -68,7 +70,13 @@ class ArticleCarbon implements Timestampable
     private $author;
 
     /**
-     * @var \DateTime|\Carbon\Carbon|null
+     * @var \DateTime|Carbon|null
+     *
+     * @Gedmo\Timestampable(on="create")
+     * @ORM\Column(name="created", type="date")
+     */
+    /**
```
Comment (phansys) [inline tests/Gedmo/Timestampable/Fixture/ArticleCarbon.php:78]: Check these cases.

File: src/Translatable/TranslatableListener.php [hunk 3]
```diff
@@ -337,8 +339,7 @@ public function getListenerLocale()
      * @param ClassMetadata $meta
      * @param object        $om
      *
-     * @throws \Gedmo\Exception\RuntimeException if language or locale property is not
-     *                                           found in entity
```
Comment (phansys) [inline src/Translatable/TranslatableListener.php:341]: Check this case.

File: src/Uploadable/UploadableListener.php [hunk 4]
```diff
@@ -89,7 +90,7 @@ public function __construct(MimeTypeGuesserInterface $mimeTypeGuesser = null)
     {
         parent::__construct();
 
-        $this->mimeTypeGuesser = $mimeTypeGuesser ? $mimeTypeGuesser : new MimeTypeGuesser();
+        $this->mimeTypeGuesser = $mimeTypeGuesser ?: new MimeTypeGuesser();
```
Comment (franmomu) [inline src/Uploadable/UploadableListener.php:93]:
```suggestion
        $this->mimeTypeGuesser = $mimeTypeGuesser ?? new MimeTypeGuesser();
```

File: src/Uploadable/UploadableListener.php [hunk 5]
```diff
@@ -260,7 +261,7 @@ public function processFile(AdapterInterface $ea, $object, $action)
 
         if ($config['allowedTypes'] || $config['disallowedTypes']) {
             $ok = $config['allowedTypes'] ? false : true;
-            $mimes = $config['allowedTypes'] ? $config['allowedTypes'] : $config['disallowedTypes'];
+            $mimes = $config['allowedTypes'] ?: $config['disallowedTypes'];
```
Comment (franmomu) [inline src/Uploadable/UploadableListener.php:264]: I would like to check the possible values of `$config['allowedTypes']` and `$config['disallowedTypes']`, I think I can check this later.

**Review by franmomu — COMMENTED** [review]
I guess this could add conflicts with https://github.com/doctrine-extensions/DoctrineExtensions/pull/2447, if next release is a minor one, I would merge https://github.com/doctrine-extensions/DoctrineExtensions/pull/2447 first

File: src/Uploadable/UploadableListener.php (hunk 5 above)
Comment (phansys) [reply src/Uploadable/UploadableListener.php:264]: Yes, I agree. These cases deserve a new round:
https://github.com/doctrine-extensions/DoctrineExtensions/blob/bf8b828f1f68e0e0b6cb2a459051c08214285c8e/tests/Gedmo/Uploadable/Mapping/ValidatorTest.php#L221-L222

**Review by franmomu — APPROVED** [review]

**Comment by franmomu** [comment]
thanks @phansys!

**Pull request merged**